client/client_test.go

```diff
@@ -1893,6 +1895,74 @@ func testUser(t *testing.T, sb integration.Sandbox) {
 	checkAllReleasable(t, c, sb, true)
 }
 
+func testPreventsMultipleExportWithSameExporter(t *testing.T, sb integration.Sandbox) {
+	integration.SkipIfDockerd(t, sb, "multiple exporters")
+	requiresLinux(t)
+
+	c, err := New(sb.Context(), sb.Address())
+	require.NoError(t, err)
+	defer c.Close()
+
+	def, err := llb.Image("busybox").Marshal(context.TODO())
+	require.NoError(t, err)
+
+	destDir1, destDir2 := t.TempDir(), t.TempDir()
+
+	_, err = c.Solve(sb.Context(), def, SolveOpt{
+		Exports: []ExportEntry{
+			{
+				Type:      ExporterLocal,
+				OutputDir: destDir1,
+			},
+			{
+				Type:      ExporterLocal,
+				OutputDir: destDir2,
+			},
+		},
+	}, nil)
+	require.Errorf(t, err, "using multiple ExporterLocal is not supported")
```

Comment:
Why is this not allowed?

Reply:
I guess if we would want to support it, it would be a client-side only change. No point in actually transferring files from daemon 2 times but client-side could write it to multiple locations.

Reply:
Yes, this makes sense. I failed to see how to make the client duplicate the output to multiple destinations easily so I need to study this a bit more (I mean the `fsSyncTarget` implementation if I'm not mistaken).

Reply:
You can leave this one out for now if it gets complicated.